src/main/java/pl/ug/virtualofficebackend/domain/user/internal/UserRepository.java

```diff
@@ -0,0 +1,7 @@
+package pl.ug.virtualofficebackend.domain.user.internal;
+
+import org.springframework.data.repository.CrudRepository;
+import pl.ug.virtualofficebackend.domain.user.entity.User;
+
+public interface UserRepository extends CrudRepository<User, Long> {
```

Comment:
Sam podzial internal/boundary jest spoko ale imo lepiej i czytelniej byloby uzycie entity/service/repository

Reply:
To wyjaśniłem wyżej

Reply:
plus potem możliwe, że bedzie trzeba tez dodać jakies klasy walidujace wiec ten warto ustalic gdzie takie rzeczy chcemy trzymac

Reply:
Właśnie klasy walidujące, repozytoria itp trzymalibyśmy w internal, bo to raczej nie jest coś takiego, z czego inne serwisypowinny korzystać.
No chyba, że będziemy potrzebować jakichś uniwersalnych walidatorów, które będą potrzebne w wielu miejscach. To wtedy warto je wydzielić do oddzielnego pakietu: domain/validator, czy jakoś tak(zamiast domain/user/internal)

Reply:
Imo trzymanie walidatorow razem z repo i implementacja serwisu wprowadzi spory bajzel. Lepiej wydzieli pakiety na service/repo/validator potem jak bedzie wiecej plików moze byc ciezko sie odnalezc. Mozna nawet podzielic to na foldery w srodku internal

Reply:
Hm, możemy też tak zrobić, ale pytanie, czy to też ma sens. W tym przypadku ma, ale czy dla zwykłego cruda na danej tabeli, gdzie nie bedzie większej logiki prócz jednorazowej walidacji, jest sens tworzenie oddzielnego pakietu dla jednego walidatora, jednej implementacji serwisu, jednego repozytorium?

Reply:
Tutaj tez troche zle napisalem  - 
user/service
user/repo
user/validator

bardziej chodzilo mi o to zeby to wymienic

Reply:
To może już prędzej robić to na bieżąco, gdy odczujemy potrzebę takiego wydzielenia?

Reply:
Mozemy sie tak umówić, ale jesli chcemy zawsze sie ograniczac do jednego entity to w sumie rzeczywiscie nie bedzie to potrzebne